clarifai.go

```diff
@@ -37,7 +37,7 @@ func NewClarifaiClient() *ClarifaiClient {
 }
 
 // Check whether the client still has access rights to Clarifai
-func (client *ClarifaiClient) isAccess() bool {
+func isCLFAccessible(client *ClarifaiClient) bool {
```

Comment:
1. Appending methods to a struct is not necessarily an OO feature and is actually very fundamental Go. I am almost certain you'll see it in any Go project you'll take a look at. In fact, when we're doing `http.Get` in our code we're actually calling the Get method on the default http client struct (see here: https://golang.org/pkg/net/http/#Client.Get). 

2. It has absolutely no effect on whether we're using frameworks or not, as it is not a parameter. You can easily register handler functions that are appended to a struct. Example: in Gin: `func (s *MyStruct) someFunc(c *gin.Context)` is a valid handler, which you can register like that: `app.GET("/endpoint", myStruct.someFunc)`. In standard net/http: `func (s *Struct) someFunc(w http.ResponseWriter, r *http.Request)` is a valid handler, which you can register like that: `http.HandleFunc("/endpoint", myStruct.someFunc)`

I was looking at some API clients when I first learned Go and they pretty much use the same approach. For example: https://github.com/fsouza/go-dockerclient/blob/master/client.go


This is my opinion though. Since no one else gave an opinion, it's up to you to choose what to do. I'm 100% OK with it if you choose to leave it as it, I just prefer the other approach